doc/source/user_guide/missing_data.rst

```diff
@@ -920,3 +920,27 @@ filling missing values beforehand.
 
 A similar situation occurs when using Series or DataFrame objects in ``if``
 statements, see :ref:`gotchas.truth`.
+
+NumPy ufuncs
+------------
+
+:attr:`pandas.NA` implements NumPy's ``__array_ufunc__`` protocol. Most ufuncs
+work with ``NA``, and generally return ``NA``:
+
+.. ipython:: python
+
+   np.log(pd.NA)
+   np.add(pd.NA, 1)
```

Comment:
Didn't yet check in detail the PR, but does this then also special case for np.power / np.divide ? (similar to what we have been discussing lately for eg IntegerArray)

Reply:
Yes. They dispatch to the dunder op

```python
In [20]: np.power(pd.NA, np.array([0, 1]))
Out[20]: array([1, NA], dtype=object)
```